build.sbt

```diff
@@ -54,7 +54,7 @@ lazy val microservice =
     )
     .settings(
       scalacOptions += "-Wconf:src=routes/.*:s",
-      scalacOptions += "-Wconf:cat=unused-imports&src=txt/.*:s"
+      scalacOptions += "-Wconf:msg=unused-imports&src=txt/.*:s"
```

Comment:
Is this correct? am sure we will still be getting warnings for unused imports in txt files with above. as  `msg` and `cat` are different options and I believe it should be more about tidying up to use `-Wunused` instead

Reply:
I was following the migration guide:

> -Wconf now works with Scala 3.3.4 (also 3.5.0) - but the syntax has changed slightly.
Instead of using `cat` you can use `msg` so for example
scalacOptions += "-Wconf:cat=unused-imports&src=html/.*:s"
will become
scalacOptions += "-Wconf:msg=unused import&src=html/.*:s"

but, yes, this warning still shows. Can we jump on a call to resolve?

Reply:
Nevermind, need to get an eye test!

Reply:
Something like this should work
```
      scalacOptions += "-Wconf:src=routes/.*:s,src=txt/.*:s",
      scalacOptions += "-Wunused:imports"
```